.dependabot/config.yml

```diff
@@ -0,0 +1,10 @@
+version: 1
+update_configs:
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "daily"
+    allowed_updates:
```

Comment:
Does this mean it will only send updates for our own stuff. Wouldn't it be better to do updates for anything, i.e. the vscode language server also?

Reply:
I am open to having dependabot watch all dependencies.  I believe @nrkruk had some reservations about this.